benchmarks/microbenchmarks/test/benchmark_config.yml

```diff
@@ -0,0 +1,19 @@
+# Sample configuration for inference kernel benchmarks
+quantizations:
+  - "baseline"
+  - "int8wo"
+  - "int4wo-128"
+  - "int4wo-128-hqq"
+output_dir: "benchmarks/microbenchmarks/test/results"  # Directory for results and plots
+model_params:
+  matrix_shapes:
+    - name: "custom"
+      shapes: [
+        [1024, 1024, 1024],  # [m, k, n]
+        [2048, 4096, 1024],
+        [4096, 4096, 1024]
+      ]
+  precision: "torch.bfloat16"
```

Comment:
nit: there are many different precisions in this context.  Maybe this can be named more specifically, such as "high_precision_dtype"?